tests/integration/cmd_pref_config_test.go

```diff
@@ -234,25 +234,36 @@ var _ = Describe("odo preference and config command tests", func() {
 	})
 
 	Context("when viewing local config without logging into the OpenShift cluster", func() {
-		It("should list config successfully", func() {
-			helper.Cmd("odo", "create", "--s2i", "nodejs", "--git", "https://github.com/openshift/nodejs-ex", "--project", commonVar.Project, "--context", commonVar.Context).ShouldPass()
-			helper.Cmd("odo", "config", "set", "--env", "hello=world", "--context", commonVar.Context).ShouldPass()
-			kubeconfigOld := os.Getenv("KUBECONFIG")
-			os.Setenv("KUBECONFIG", "/no/such/path")
-			configValue := helper.Cmd("odo", "config", "view", "--context", commonVar.Context).ShouldPass().Out()
-			helper.MatchAllInOutput(configValue, []string{"hello", "world"})
-			os.Setenv("KUBECONFIG", kubeconfigOld)
+		var ocRunner helper.OcRunner
+		var token string
+		BeforeEach(func() {
+			helper.CopyExample(filepath.Join("source", "nodejs"), commonVar.Context)
+			helper.Cmd("odo", "create", "--s2i", "nodejs", "nodejs", "--project", commonVar.Project, "--context", commonVar.Context).ShouldPass()
+			ocRunner = helper.NewOcRunner("oc")
+			token = ocRunner.GetToken()
+			ocRunner.Logout()
 		})
-
-		It("should set config variable without logging in", func() {
-			helper.Cmd("odo", "create", "--s2i", "nodejs", "--project", commonVar.Project, "--context", commonVar.Context).ShouldPass()
-			kubeconfigOld := os.Getenv("KUBECONFIG")
-			os.Setenv("KUBECONFIG", "/no/such/path")
-			helper.Cmd("odo", "config", "set", "--force", "--context", commonVar.Context, "Name", "foobar").ShouldPass()
-			configValue := helper.Cmd("odo", "config", "view", "--context", commonVar.Context).ShouldPass().Out()
-			Expect(configValue).To(ContainSubstring("foobar"))
-			helper.Cmd("odo", "config", "unset", "--force", "--context", commonVar.Context, "Name").ShouldPass()
-			os.Setenv("KUBECONFIG", kubeconfigOld)
+		AfterEach(func() {
+			ocRunner.LoginUsingToken(token)
+		})
+		When("user is working with a devfile component", func() {
```

Comment:
I understand that we now convert s2i comps to devfile comps, but you are creating a comp with s2i in the setup and this statement says we're testing this for a devfile component. Perhaps you can rename this statement or create a devfile comp in the setup to amend this. WDYT?